src/main/java/seedu/address/model/commons/NameContainsKeywordsPredicate.java

```diff
@@ -4,22 +4,21 @@
 import java.util.function.Predicate;
 
 import seedu.address.commons.util.StringUtil;
-import seedu.address.model.activity.Activity;
 
 /**
- * Tests that a {@code Activity}'s {@code Name} matches any of the keywords given.
+ * Tests that a {@code TravelPlanObject}'s {@code Name} matches any of the keywords given.
  */
-public class NameContainsKeywordsPredicate implements Predicate<Activity> {
+public class NameContainsKeywordsPredicate implements Predicate<TravelPlanObject> {
```

Comment:
Maybe here can also implement Predicate<TravelPlan>, then we can overload the test method with test(TravelPlan tp) as well?

Reply:
I updated it. But I don't know if it works, can help me check?